### PR TITLE
Update Text docs

### PR DESCRIPTION
**Pull request by sion-docs**
Added new sections and expanded content in existing sections, please check for technical accuracy.

### CODE REVIEW

File: packages/core/stories/text/text.doc.stories.mdx [hunk 1]
```diff
@@ -32,22 +32,21 @@ We've created a base Text component (that contains our key typography styles and
...
+## Highlighting
 
-## Props
+Text provides a default highlight color. Use the token `text-background-selected` to indicate a selection of text using a cursor, in both light and dark modes. 
 
-<ArgsTable of={Text} />
+A selection of text should be able to be highlighted with the default highlight color applied, this includes text within input fields.
 
-## Features
+The highlight function can also be used to highlight images, or other content, alongside text.
```
Comment (joshwooding) [inline packages/core/stories/text/text.doc.stories.mdx:44]: I don't think we need to mention this here

File: packages/core/stories/text/text.doc.stories.mdx [hunk 2]
```diff
@@ -60,9 +59,13 @@ Use `variant` prop when you want to change the text color. Available options are
   </Story>
 </Canvas>
 
-### Emphasis
+## Emphasis
+
+There are three emphasis levels across our text styles that can be used to highlight or lowlight body text and headings - the default level with no emphasis, `strong` to emphasise text/content and `small` to de-emphasise text/content.
```
Comment (joshwooding) [inline packages/core/stories/text/text.doc.stories.mdx:64]:
```suggestion
There are three emphasis levels across our text styles that can be used to highlight or lowlight body text and headings - the default level with no emphasis, `strong` to emphasize text/content and `small` to de-emphasize text/content. 
```

File: packages/core/stories/text/text.doc.stories.mdx (hunk 1 above)
Comment (sion-docs) [reply packages/core/stories/text/text.doc.stories.mdx:44]: OK

File: packages/core/stories/text/text.doc.stories.mdx [hunk 3]
```diff
@@ -32,22 +32,13 @@ We've created a base Text component (that contains our key typography styles and
 
 You can also create your own text components using the props and CSS variables to configure them.
 
-# API
+# Features
```
Comment (joshwooding) [inline packages/core/stories/text/text.doc.stories.mdx:35]:
```suggestion
## Features
```

File: packages/core/stories/text/text.doc.stories.mdx [hunk 4]
```diff
@@ -32,22 +32,13 @@ We've created a base Text component (that contains our key typography styles and
...
 
-```
-import { Text, H1, H2, H3, H4, Label, Display1, Display2, Display3 } from "@jpmorganchase/uitk-core";
-```
-
-## Props
-
-<ArgsTable of={Text} />
+## Variant
```
Comment (joshwooding) [inline packages/core/stories/text/text.doc.stories.mdx:37]:
```suggestion
### Variant
```
But also, in Button we have a title Button variants instead. We should be consistent.

File: packages/core/stories/text/text.doc.stories.mdx [hunk 5]
```diff
@@ -104,8 +103,26 @@ By using the `maxRows` prop your text component will be truncated and will displ
   </Story>
 </Canvas>
 
+# Configuring Text
+## API
```
Comment (joshwooding) [inline packages/core/stories/text/text.doc.stories.mdx:107]:
```suggestion
### API
```

File: packages/core/stories/text/text.doc.stories.mdx [hunk 6]
```diff
@@ -104,8 +103,26 @@ By using the `maxRows` prop your text component will be truncated and will displ
...
+
+```
+import { Text, H1, H2, H3, H4, Label, Display1, Display2, Display3 } from "@jpmorganchase/uitk-core";
+```
+
+## Props
```
Comment (joshwooding) [inline packages/core/stories/text/text.doc.stories.mdx:113]:
```suggestion
### Props
```

File: packages/core/stories/text/text.doc.stories.mdx [hunk 7]
```diff
@@ -104,8 +103,26 @@ By using the `maxRows` prop your text component will be truncated and will displ
...
+
+<ArgsTable of={Text} />
+
 ## CSS Class
```
Comment (joshwooding) [inline packages/core/stories/text/text.doc.stories.mdx:117]:
```suggestion
### CSS Class
```
Should this be CSS Class or Classes

**Review by joshwooding — CHANGES_REQUESTED** [review]

File: packages/core/stories/text/text.doc.stories.mdx (hunk 7 above)
Comment (sion-docs) [reply packages/core/stories/text/text.doc.stories.mdx]: Classes makes more sense

**Pull request merged**